package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uapi-json",
-  "version": "1.14.0",
```

Comment:
Please change minor instead of patch as this change is not backwards compatible and all users will start to receive other information for the same calls